package.json

```diff
@@ -7,18 +7,18 @@
     "build": "encore production"
   },
   "dependencies": {
-    "awesomplete": "^1.1.2",
-    "clipboard": "^1.7.1",
-    "promise-polyfill": "^7.1.2",
-    "raven-js": "^3.23.1",
-    "tooltip.js": "^1.1.7",
-    "whatwg-fetch": "^2.0.4"
+    "awesomplete": "^1.1.3",
+    "clipboard": "^2.0.1",
+    "promise-polyfill": "^8.1.0",
+    "raven-js": "^3.27.0",
+    "tooltip.js": "^1.3.0",
+    "whatwg-fetch": "^3.0.0"
   },
   "devDependencies": {
-    "@symfony/webpack-encore": "^0.19.0",
-    "bootstrap": "^4.0.0",
-    "node-sass": "^4.7.2",
-    "sass-loader": "^6.0.6",
-    "webpack-notifier": "^1.5.1"
+    "@symfony/webpack-encore": "^0.20.1",
+    "bootstrap": "^4.1.3",
+    "node-sass": "^4.9.4",
+    "sass-loader": "^6.0.7",
```

Comment:
it would be great to switch to `dart-sass`, but i think `webpack-encore` itself only supports `node-sass`.

see: https://github.com/symfony/webpack-encore/issues/422

Reply:
👍